### PR TITLE
DOC-5359 log export aws consistency pass

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-5359

- Converts `## Limitations` to FAQ style, following style presented in CMEK docs.
- Updates all `curl` invocations to use the `--request` and `--url` flags to match existing CMEK docs.
- Added Dedicated support to `logging` category on API Support Policy reference page, for both v22.1 and newly-cut v22.2 docs.
- (Updating language from `Private Preview` to `preview` already accomplished [previous to this PR](https://github.com/cockroachdb/docs/pull/14827).)

Notes:
- Opting not to match external `json` conf file approach in CMEK step 4, see comment to Jira ticket.
- Many consistency requests were merged in time for GA release of this feature. If going through comments made to #[14739](https://github.com/cockroachdb/docs/pull/14739), you'll see most were made already; remaining asks that did not make it are addressed by this ticket.

[export-logs.md](https://deploy-preview-14881--cockroachdb-docs.netlify.app/docs/cockroachcloud/export-logs.html) | [v22.2/api-support-policy.md](https://deploy-preview-14881--cockroachdb-docs.netlify.app/docs/dev/api-support-policy.html#apis) | [v22.1/api-support-policy.md](https://deploy-preview-14881--cockroachdb-docs.netlify.app/docs/stable/api-support-policy.html#apis)

### CODE REVIEW

**Review by thtruo — APPROVED** [review]
Nice @andf-crl - this latest pass LGTM. I agree with your take that we don't need to match the `json` conf file like in the CMEK docs 

:lgtm:

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14881#-:-N9msaHQ4T8fyVkWZnY5:b4sxml1)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @andf-crl)

**Comment by andf-crl** [comment]
Thanks Tommy. @abhinavg6 new FAQ items look OK to you?

File: cockroachcloud/export-logs.md [hunk 1]
```diff
@@ -179,13 +183,27 @@ Where:
 - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
 
 
-## Limitations
+## {{ site.data.products.dedicated }} log export Frequently Asked Questions (FAQ)
+
+### Is it possible to configure exported logs to be redacted at source?
+
+Logs exported in this fashion retain [`redactable`](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) markers, but are **not** themselves redacted. Do not use this feature with sensitive log messages that you do not wish to export.
```
Comment (abhinavg6) [inline cockroachcloud/export-logs.md:190]:
```suggestion
Logs exported in this fashion retain [`redactable`](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) markers, but are **not** themselves redacted. Based on your needs, you could optionally redact sensitive data in the logs at your end by making use of `redactable` markers, before the logs are consumed by your team.
```

File: cockroachcloud/export-logs.md [hunk 2]
```diff
@@ -179,13 +183,27 @@ Where:
...
+
+### Is it possible to configure multiple log export configurations to send different log channels to different log groups in AWS CloudWatch?
+
+No, only one log export configuration is currently possible per cluster.
+
+### For a multi-region cluster, are the logs exported to one CloudWatch region?
```
Comment (abhinavg6) [inline cockroachcloud/export-logs.md:196]:
```suggestion
### For a multi-region cluster, are the logs from all regions exported to one CloudWatch region?
```

File: cockroachcloud/export-logs.md [hunk 3]
```diff
@@ -179,13 +183,27 @@ Where:
...
+
+Yes, logs for each region in your cluster are exported to the corresponding CloudWatch region configured for your account. Ensure that the target AWS CloudWatch [log group](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/iam-access-control-overview-cwl.html#CWL_ARN_Format) is configured with the same name in all target regions, and that the [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) you are using has permission to access each regional log group.
```
Comment (abhinavg6) [inline cockroachcloud/export-logs.md:198]:
```suggestion
No, logs for each region in your cluster are exported to the corresponding CloudWatch region configured for your account. Ensure that the target AWS CloudWatch [log group] is configured with the same name in all target regions, and that the [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) you are using has permission to access each regional log group.
```

File: cockroachcloud/export-logs.md [hunk 4]
```diff
@@ -179,13 +183,27 @@ Where:
...
-- Only one log export configuration is possible per cluster.
-- A cluster log configuration is shared by all nodes in all regions in the cluster. 
-- Only the following CockroachDB [log channels](/docs/{{site.versions["stable"]}}/logging-overview.html#logging-channels) are supported for export in this manner: `SESSIONS`,`OPS`, `HEALTH`, `STORAGE`, `SQL_SCHEMA`, `USER_ADMIN`, `PRIVILEGES`, `SENSITIVE_ACCESS`, `SQL_EXEC`, and `SQL_PERF`. Other log channels are not exportable from {{ site.data.products.dedicated }}.
-- To export the [SQL Audit Log](/docs/{{site.versions["stable"]}}/sql-audit-logging.html) via the `SENSITIVE_ACCESS` log channel, you must additionally enable audit logging for the desired tables using the [`ALTER TABLE ...EXPERIMENTAL_AUDIT`](/docs/{{site.versions["stable"]}}/experimental-audit.html) statement.
+Yes, the [SQL Audit Log](/docs/{{site.versions["stable"]}}/sql-audit-logging.html) is exported via the `SENSITIVE_ACCESS` log channel by default, as long as you have first enabled audit logging on desired tables using the [`ALTER TABLE ...EXPERIMENTAL_AUDIT`](/docs/{{site.versions["stable"]}}/experimental-audit.html) statement.
```
Comment (abhinavg6) [inline cockroachcloud/export-logs.md:206]:
```suggestion
Yes, the [SQL Audit Log](/docs/{{site.versions["stable"]}}/sql-audit-logging.html) is exported via the `SENSITIVE_ACCESS` log channel by default, as long as you have previously enabled audit logging on desired tables using the [`ALTER TABLE ...EXPERIMENTAL_AUDIT`](/docs/{{site.versions["stable"]}}/experimental-audit.html) statement.
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]

**Comment by andf-crl** [comment]
TYFR @abhinavg6 , changes made. PTAL.

**Review by abhinavg6 — APPROVED** [review]

**Comment by andf-crl** [comment]
@MichaelTrestman ready for review! This was the feature we released on a timetable (Aug 10) and skipped copy review. As you have time, I would very much appreciate a review pass on the whole `export-logs.md` page. Thanks so much!

File: cockroachcloud/export-logs.md [hunk 5]
```diff
@@ -179,13 +183,27 @@ Where:
 - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
 
 
-## Limitations
+## {{ site.data.products.dedicated }} log export Frequently Asked Questions (FAQ)
+
+### Is it possible to configure exported logs to be redacted at source?
+
+Logs exported in this fashion retain [`redactable`](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) markers, but are **not** themselves redacted. Based on your needs, you could optionally redact sensitive data in the logs at your end by making use of `redactable` markers, before the logs are consumed by your team.
```
Comment (MichaelTrestman) [inline cockroachcloud/export-logs.md:190]: "at your end" meaning you need to put another proxy application in between CRDB and whatever "team" was going to consume "your logs", in order to do the redaction? that is more non-trivial than this makes it sound, no?
You could do that with Satori I think, maybe link to the Satori integration page if that is correct? In any case I'd suggest fleshing out "at your end" a bit more in a way that is helpful for decision-making/estimating the complexity of what is involved.

**Review by MichaelTrestman — APPROVED** [review]
LGTM, other than (optionally) that spot where I noted clarification could be helpful.

**Comment by andf-crl** [comment]
@MichaelTrestman thanks so much; I'll clarify!

**Pull request merged**